src/tokens.md

```diff
@@ -337,9 +337,9 @@ b"\\x52"; br"\x52";                  // \x52
 > **<sup>Lexer</sup>**\
 > C_STRING_LITERAL :\
 > &nbsp;&nbsp; `c"` (\
-> &nbsp;&nbsp; &nbsp;&nbsp; ~\[`"` `\` _IsolatedCR_]\
-> &nbsp;&nbsp; &nbsp;&nbsp; | BYTE_ESCAPE\
-> &nbsp;&nbsp; &nbsp;&nbsp; | UNICODE_ESCAPE\
+> &nbsp;&nbsp; &nbsp;&nbsp; ~\[`"` `\` _IsolatedCR_ _ZeroByte_]\
```

Comment:
ZeroByte doesn't seem to be defined anywhere, and I think could be confused with DIGIT ZERO. Could we use a different term here? Perhaps just Null?

```suggestion
> &nbsp;&nbsp; &nbsp;&nbsp; ~\[`"` `\` _IsolatedCR_ _Null_]\
```

Reply:
I didn't define ZeroByte because IsolatedCR also isn't defined. I have changed ZeroByte to Nul. (Not Null, because that tends to be used for the zero pointer while Nul tends to be used for the zero character.)

Reply:
For what it's worth, the `IsolatedCR` definition is hiding in the "Comments" chapter.

Reply:
I've changed it from "Nul" to "NUL", because "NUL" is defined earlier in the sentence "The null escape is the character U+0030 (0) and denotes the Unicode value U+0000 (NUL)." which I hope is good enough?

Reply:
Sounds good. I don't know if the world has an agreement if it is NULL or NUL. They seem to be used interchangeably, and I don't think there is any ambiguity what either spellings mean here, which means I think it is largely a matter of personal preference.

Reply:
Thinking some more, I suspect "null" is the proper word, and NUL is an ASCII-ism (because the control chars all have two or three letter abbreviations) that has crept into ordinary use, including Unicode names.